OutSystems/outsystems-datagrid-reactive
# ROU-3217: OnCellValueChangeEvent of Data Grid not fired when two columns have the same binding

This PR is for OnCellValueChangeEvent of Data Grid to be fired even when two columns have the same binding

### What was happening
* The OnCellValueChangeEvent was not being fired when I have two columns in the data grid binding to the same attribute in the database. I set a client action as the handler to the event on one of the columns. When I changed the value of the cell of that column, the client action does not get executed. However, when I remove the other column with the same binding, the event is fired and handled by the client action as it should be.

### What was done
* Change _triggerEventsFormColumn to use column index instead of column binding as input parameter

### Test Steps
1. Add a DataGrid to the screen
2. Config multiple columns with the same binding; 
3. Add custom events to some of the columns on OnCellValueChange;
4. Check if the event feedback is the expected

### Checklist
